.circleci/config.yml

```diff
@@ -17,6 +17,54 @@
 # Check https://circleci.com/docs/2.0/language-python/ for more details
 #
 version: 2
+
+check_services_status: &check_services_status
+  command: |
+    set -x
+
+    sleep 10  # Give the services enough time to start (or fail)
+
+    for c in as1301 as1303 as1305; do
+      docker-compose exec $c /bin/bash -c \
+        '! systemctl is-failed scion-*'
+    done
+
+check_scion_connections: &check_scion_connections
+  command: |
+    set -x
+
+    # Wait for beacons
+    docker-compose exec -T as1303 /bin/bash -c \
+      'until [ `grep "Registered beacons" -s -r /var/log/scion/ | wc -l` -ge 3 ]; do sleep 0.1; done;'
+    docker-compose exec -T as1305 /bin/bash -c \
+      'until [ `grep "Registered beacons" -s -r /var/log/scion/ | wc -l` -ge 3 ]; do sleep 0.1; done;'
```

Comment:
Should we clear the log directory (maybe create a backup for debugging) at some point to avoid false positives after the restart for the update test?

Reply:
Good point, hadn't thought about that one

Reply:
Thinking about it:
>Should we clear the log directory

I think that's not what we want, since when the package gets updated logs are also not cleared

>avoid false positives after the restart

What we want is checking for fresh beacons to make sure that beaconing has started and beacons are propagated properly.